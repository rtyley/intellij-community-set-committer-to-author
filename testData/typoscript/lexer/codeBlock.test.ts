myObject {
field = title
   ifEmpty {  ignored text
       data = leveltitle:0  } bracket is ignored
     [i = 5]
    }    some ignored text
}
} this brace is also ignored by parser

second.object{
{ ignored opening bracket
 .10 = left
 prop < test
 text(
  } ignored bracket
 )
}

third.object{
  do.not.ignore{
    i=5
  }
}