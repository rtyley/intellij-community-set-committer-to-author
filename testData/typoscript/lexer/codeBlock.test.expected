OBJECT_PATH_ENTITY ('myObject')
WHITE_SPACE (' ')
CODE_BLOCK_OPEN ('{')
WHITE_SPACE ('
')
OBJECT_PATH_ENTITY ('field')
WHITE_SPACE (' ')
ASSIGN ('=')
WHITE_SPACE (' ')
ASSIGNMENT_VALUE ('title')
WHITE_SPACE ('
   ')
OBJECT_PATH_ENTITY ('ifEmpty')
WHITE_SPACE (' ')
CODE_BLOCK_OPEN ('{')
IGNORED_TEXT ('  IGNORED_TEXT')
WHITE_SPACE ('
       ')
OBJECT_PATH_ENTITY ('data')
WHITE_SPACE (' ')
ASSIGN ('=')
WHITE_SPACE (' ')
ASSIGNMENT_VALUE ('leveltitle:0  } bracket is ignored')
WHITE_SPACE ('
     ')
CONDITION ('[i = 5]')
WHITE_SPACE ('
    ')
CODE_BLOCK_CLOSE ('}')
IGNORED_TEXT ('    some IGNORED_TEXT')
WHITE_SPACE ('
')
CODE_BLOCK_CLOSE ('}')
WHITE_SPACE ('
')
CODE_BLOCK_CLOSE ('}')
IGNORED_TEXT (' this brace is also ignored by parser')
WHITE_SPACE ('

')
OBJECT_PATH_ENTITY ('second')
OBJECT_PATH_SEPARATOR ('.')
OBJECT_PATH_ENTITY ('object')
CODE_BLOCK_OPEN ('{')
WHITE_SPACE ('
')
IGNORED_TEXT ('{ ignored opening bracket')
WHITE_SPACE ('
 ')
OBJECT_PATH_SEPARATOR ('.')
OBJECT_PATH_ENTITY ('10')
WHITE_SPACE (' ')
ASSIGN ('=')
WHITE_SPACE (' ')
ASSIGNMENT_VALUE ('left')
WHITE_SPACE ('
 ')
OBJECT_PATH_ENTITY ('prop')
WHITE_SPACE (' ')
COPY ('<')
WHITE_SPACE (' ')
OBJECT_PATH_ENTITY ('test')
WHITE_SPACE ('
 ')
OBJECT_PATH_ENTITY ('text')
MULTILINE_OPEN ('(')
WHITE_SPACE ('
  ')
MULTILINE_VALUE ('} ignored bracket')
WHITE_SPACE ('
 ')
MULTILINE_CLOSE (')')
WHITE_SPACE ('
')
CODE_BLOCK_CLOSE ('}')
WHITE_SPACE ('

')
OBJECT_PATH_ENTITY ('third')
OBJECT_PATH_SEPARATOR ('.')
OBJECT_PATH_ENTITY ('object')
CODE_BLOCK_OPEN ('{')
WHITE_SPACE ('
  ')
OBJECT_PATH_ENTITY ('do')
OBJECT_PATH_SEPARATOR ('.')
OBJECT_PATH_ENTITY ('not')
OBJECT_PATH_SEPARATOR ('.')
OBJECT_PATH_ENTITY ('ignore')
CODE_BLOCK_OPEN ('{')
WHITE_SPACE ('
    ')
OBJECT_PATH_ENTITY ('i')
ASSIGN ('=')
ASSIGNMENT_VALUE ('5')
WHITE_SPACE ('
  ')
CODE_BLOCK_CLOSE ('}')
WHITE_SPACE ('
')
CODE_BLOCK_CLOSE ('}')